tests/Areas/AppConfig/UnitTests/KeyValue/KeyValueSetCommandTests.cs

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using System.Text.Json;
+using AzureMcp.Areas.AppConfig.Commands.KeyValue;
+using AzureMcp.Areas.AppConfig.Models;
+using AzureMcp.Areas.AppConfig.Services;
+using AzureMcp.Models.Command;
+using AzureMcp.Options;
+using AzureMcp.Tests.Models.AppConfig;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using Xunit;
+
+namespace AzureMcp.Tests.Areas.AppConfig.UnitTests.KeyValue;
+
+[Trait("Area", "AppConfig")]
+public class KeyValueSetCommandTests
+{
+    private readonly IServiceProvider _serviceProvider;
+    private readonly IAppConfigService _appConfigService;
+    private readonly ILogger<KeyValueSetCommand> _logger;
```

Comment:
I think this is the only test class that did not get that change where you added

```C#
private readonly KeyValueShowCommand _command;
private readonly CommandContext _context;
private readonly Parser _parser;
```

Let's get this one aligned with the rest.